main.py

```diff
@@ -74,24 +74,21 @@ def bootstrap(
 
 
 def train_dir(
-        working_dir: 'tf.estimator working directory.',
         chunk_dir: 'Directory where training chunks are.',
         model_save_path: 'Where to export the completed generation.'):
     tf_records = sorted(gfile.Glob(os.path.join(chunk_dir, '*.tfrecord.zz')))
     tf_records = tf_records[-1 * (WINDOW_SIZE // EXAMPLES_PER_RECORD):]
 
-    train(working_dir, tf_records, model_save_path)
+    train(tf_records, model_save_path)
 
 
-def train(
-        working_dir: 'tf.estimator working directory.',
-        tf_records: 'list of files of tf_records to train on',
+def train(tf_records: 'list of files of tf_records to train on',
         model_save_path: 'Where to export the completed generation.'):
     print("Training on:", tf_records[0], "to", tf_records[-1])
     with utils.logged_timer("Training"):
-        dual_net.train(working_dir, *tf_records)
+        dual_net.train(*tf_records)
     print("== Training done.  Exporting model to ", model_save_path)
-    dual_net.export_model(working_dir, model_save_path)
+    dual_net.export_model(flags.FLAGS.model_dir, model_save_path)
```

Comment:
you could update the export_model() function to use flags instead of a function arg